### ISSUE TITLE
Debian / Ubuntu builds 404

### ISSUE THREAD

**Issue by fidelix** [Gurkengewuerz/KShare#13]
https://app.circleci.com/github/Gurkengewuerz/KShare/pipelines?branch=dev
https://app.circleci.com/github/Gurkengewuerz/KShare/pipelines?branch=master

Both give 404 even after I created a circleci account (it asked me to).

**Comment by Gurkengewuerz** [comment]
Hmmm strange. The project should be public. You can get the current master branch [here](https://github.com/Gurkengewuerz/KShare/files/4664468/kshare_v5.0.4.zip).

**Comment by fidelix** [comment]
```sudo dpkg -i kshare_v5.0.4.deb 
Selecting previously unselected package kshare.
(Reading database ... 213398 files and directories currently installed.)
Preparing to unpack kshare_v5.0.4.deb ...
Unpacking kshare (5.0.4) ...
dpkg: dependency problems prevent configuration of kshare:
 kshare depends on libavcodec57 (>= 3.2.0); however:
  Package libavcodec57 is not installed.
 kshare depends on libavformat57 (>= 3.2.0); however:
  Package libavformat57 is not installed.
 kshare depends on libswscale4 (>= 3.2.0); however:
  Package libswscale4 is not installed.
 kshare depends on libavutil55 (>= 3.2.0); however:
  Package libavutil55 is not installed.

dpkg: error processing package kshare (--install):
 dependency problems - leaving unconfigured
Processing triggers for desktop-file-utils (0.24-1ubuntu2) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Errors were encountered while processing:
 kshare
```

I'm on Kubuntu 20.04.
Seems I have libavcodec58, libavformat58, etc.
I'll try to install both versions somehow.

EDIT: Doesn't seem possible, I entered dependency hell.

**Comment by Gurkengewuerz** [comment]
fixed with 4f9b25190b3e93e82d7231634f1bf73d1d13800d

The other issue is related to #14
